tests/behat/features/bootstrap/Stack.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @file
+ * Implements a simple LIFO stack
```

Comment:
Not sure what this class does that couldn't be achieved using `array_push` and `array_pop`

Reply:
Fair point. It seemed at the time like it would be. Refactored to just be an array.